### ISSUE TITLE
Product page v2 minor feedback

### ISSUE THREAD

**Issue by Hlavtox** [PrestaShop/PrestaShop#28179]
Just had another look on the new product page, found only some minor things we can improve. Otherwise, it's really intuitive, faster, cleaner. Especially the combination management!!! Good job everyone, from product team, designer, developers.

### 1 Customization
The switch for "required" has no label, I think it could be confusing for merchants that don't know what it means.
Can we add some label or is it designed somewhere?

![customization](https://user-images.githubusercontent.com/6097524/162396176-0f7ae392-51b7-48ec-8192-f92a72c6f2a4.jpg)

### 2 Availability
Availability label settings are currently on combinations tab, I think it should be moved to shipping tab, close to delivery times, because they are very closely related. Maybe it's just because this function is not finished.

![availability](https://user-images.githubusercontent.com/6097524/162396262-79135f47-64e2-4648-b6c6-25aa89a932f3.jpg)

### 3 Availability on combinations
Please, think about if it's not the best time to implement very important proposals now. So the module developers must adapt only one time for the product page. 👍 I have two clients who sell clothing and they both face the same problem. The root of this problem is that their suppliers have some sizes in stock, and some not.

- They cannot assign availability information to their combinations - https://github.com/PrestaShop/PrestaShop/issues/20536 (it's in TODO state)
- They cannot enable/disable backorders for their combinations https://github.com/PrestaShop/PrestaShop/issues/24279 (7 thumbs up on that issue)

### 4 Features - custom vs predefined
Lot of unexperienced merchants are manually typing the feature value to custom field, even if the feature value already exists. 

It makes a strange behavior in faceted search now, but that is a problem of the module. Even if that is fixed, it would be good to have it unified to a predefined value, so it's easier to debug, the database is a bit smaller, renaming the value will rename it on all products etc....

My proposal - if the custom feature value already exists, assign the predefined one on save automatically.

![cotton](https://user-images.githubusercontent.com/6097524/162397039-86e1975f-f73d-4e6c-994a-bd74f6515c47.jpg)

_____________________

Ping @MatShir @jolelievre

**Comment by eternoendless** [comment]
> The switch for "required" has no label, I think it could be confusing for merchants that don't know what it means.
> Can we add some label or is it designed somewhere?

I believe the switch component can get any label, not just on-off. Perhaps this can be solved by just changing the label.

**Comment by khouloudbelguith** [comment]
Hello @Hlavtox,

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon.
Thank you!

**Comment by jolelievre** [comment]
Hi @Hlavtox 

thanks for the feedback!

Issue 1
Yep we noticed this one as well recently with @Amoifr we plan on adding a label here Besides there are many other little things that we plan on fixing at the ned with @TristanLDD and @Julievrz to polish the UX and wording everywhere to match the final mockups

Issue 2 and 3
Actually for both technical and functional reasons We discussed with @MatShir about moving these availability preferences to the combination form meaning they will be configurable independently for each combination It's doable for the out of stock policy and available_date (already the case for available_date) But the available label are trickier they can only be handled by product for now, still need to think about this one.

Issue 4
I understand the problem you describe, but I'm not sure auto-switching from custom to real feature is the more stable way, I anticipate weird unexpected behaviour with this kind of magical changes But maybe the main problem is actually being able to create custom values which match an existing one So maybe instead add a constraint to prevent this? This would block the creation and incite using the existing one
